gitapp/lib/constants/constants.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+const primaryLight = Color(0xFF313EBB);
+const primaryDark = Color(0xFF2D239F);
+const secondaryLight = Colors.blueAccent;
+const animationtime = 700;
+const cursorColor = Colors.blueGrey;
+const NotProvided = 'Not provided 😔';
```

Comment:
Please use one common approach of naming. Lowercase, or uppercase, do not mix it.

Reply:
solved